py-packages/openverse-attribution/src/openverse_attribution/attribution.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from openverse_attribution.license import License
+
+
+def get_attribution_text(
```

Comment:
FWIW I think this function should _also_ support the `License` enum as an argument, rather than potentially having to re-enumerate all of the other fields.

Reply:
I did not get what you mean by this. Did you mean this diff to the function parameters?

```diff
- license_slug: str,
+ license: License,
```

Reply:
Yes, potentially something like `license: License | str` as the first parameter was what I was thinking! It would be a bit of overloading, but it feels odd not being able to use the library's enum for this function.

Reply:
+1 to your point, Madison.

However, let's please create an issue to sort these implementation details out elsewhere. There are a number of things we'd want to make sure are nailed down tight about this library before it was used outside our project, and it would be good to have this PR not focused on this, but the package management side of things specifically.

The main point of including it in this PR, as far as I know, is to prove the concept of shareable monorepo Python packages, not to define the actual official and publicly available API for a published attribution generation library.